docs/source/users/splash.md

```diff
@@ -0,0 +1,254 @@
+```
+
+The three sites capture wetter coastal conditions with milder temperatures (San
+Francisco), intermediate rainfall with colder temperatures (Yosemite) and arid
+conditions with extreme temperatures (Death Valley).
+
+```{code-cell} python
+# Get three sites to show time series for locations
+site_data = data.sel(sites, method="nearest")
+
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(8, 4))
+
+for idx, site_name in enumerate(site_data["site_id"].data):
+    ax1.plot(
+        site_data["time"].data,
+        site_data["tmp"].data[:, idx],
+        linewidth=0.4,
+        label=site_name,
+    )
+
+ax1.set_xlabel("Date")
+ax1.set_ylabel("Daily temperature (°C)")
+
+# Calculate the average daily precipitation within months for each site
+month_length = site_data.time.dt.days_in_month
+weights = month_length.groupby("time.month") / month_length.groupby("time.month").sum()
+ave_daily_pre = (site_data["pre"] * weights).groupby("time.month").sum(dim="time")
+
+for offset, site_name in zip([-1, 0, 1], site_data["site_id"].data):
+    ax2.bar(
+        ave_daily_pre["month"] + offset * 0.25,
+        ave_daily_pre.sel(site_id=site_name),
+        width=0.2,
+        label=site_name,
+    )
+
+ax2.set_xlabel("Month")
+ax2.set_ylabel("Average daily precipitation (mm)")
+ax2.legend()
+
+plt.tight_layout()
+```
+
+## Running the splash model
+
+### Initialising a SplashModel
+
+Before calculating water balances, you need to create a
+{class}`~pyrealm.splash.splash.SplashModel`. This takes the site data and runs all of
+the solar and evaporative calculations for the time series - none of these calculations
+rely on the soil moisture and so are calculated once when the `SplashModel` is created.
+
+::: {note}
+The `SplashModel` code currently requires that the latitude (`lat`) and elevation
+(`elv`) data have the same shape as the sunshine fraction (`sf`), temperature (`tc`) and
+precipitation (`pn`). These values are obviously constant through time - and latitude
+may well be constant across the longitude dimension for gridded data - but, at the
+moment, you need to broadcast these variables to match.
+:::
+
+```{code-cell} python
+splash = SplashModel(
+    lat=np.broadcast_to(data.lat.data[None, :, None], data.sf.data.shape),
+    elv=np.broadcast_to(data.elev.data[None, :, :], data.sf.data.shape),
+    dates=Calendar(data.time.data),
+    sf=data.sf.data,
+    tc=data.tmp.data,
+    pn=data.pre.data,
+)
+```
+
+### Estimating initial soil moisture
+
+In order to calculate water balances, you need initial values for the soil moisture.
+This data is rarely available and so the
+{meth}`~pyrealm.splash.splash.SplashModel.estimate_initial_soil_moisture` can be used
+to estimate those values.
+
+This method requires that the input data provides **at least one full year** of data. It
+works by assuming that soil moisture is a **stationary** process on an annual time
+scale: the initial soil moisture should be the same as the soil moisture at the end of
```

Comment:
I think `steady state` might be a better term here than `stationary process`. If you prefer to keep it I would change it to `soil moisture change is a stationary process`